django/contrib/gis/geos/factory.py

```diff
@@ -15,12 +17,10 @@ def fromfile(file_h):
 
     # If we get WKB need to wrap in memoryview(), so run through regexes.
     if isinstance(buf, bytes):
-        try:
+        with suppress(UnicodeDecodeError):
             decoded = buf.decode()
```

Comment:
Now I see that my comments to this effect are actually changing the semantics slightly -- previously the if statement would be skipped if `UnicodeDecodeError` happens, now they'll be run. I'm not confident if these changes are correct so I guess it's better to leave things as they were. I  wonder if try/except/else would be better than `suppress()`.